### PR TITLE
Remove hardcoded snapshot controller image references

### PR DESCRIPTION
**Pull request by ig0rsky**
**What is this PR about? / Why do we need it?**
  All the other images can be modified via changing the variable in values.yaml, except for the snapshotController. This PR fixes that.

### CODE REVIEW

File: charts/aws-ebs-csi-driver/values.yaml [hunk 1]
```diff
@@ -29,6 +29,10 @@ sidecars:
     repository: k8s.gcr.io/sig-storage/csi-node-driver-registrar
     tag: "v2.0.1"
 
+snapshotController:
+  image: k8s.gcr.io/sig-storage/snapshot-controller
```
Comment (wongma7) [inline charts/aws-ebs-csi-driver/values.yaml:33]: should say "repository:" instaed of "image:", to beconsistent with the other values

**Comment by wongma7** [comment]
/ok-to-test

thanks

File: charts/aws-ebs-csi-driver/values.yaml (hunk 1 above)
Comment (ig0rsky) [reply charts/aws-ebs-csi-driver/values.yaml]: done

**Comment by ig0rsky** [comment]
@wongma7 ready for review

**Comment by ayberk** [comment]
@ig0rsky could you please squash your commits into one? Thanks!

/lgtm

**Comment by ig0rsky** [comment]
@ayberk  done

**Comment by ayberk** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**